README.fr.md

```diff
@@ -0,0 +1,655 @@
+
+#### Ajouter des tests à un nouveau paquet
+
+Ajoutez-le à votre `tsconfig.json` :
+
+```json
+"baseUrl": "types",
+"typeRoots": ["types"],
+```
+
+Créez `types/foo/index.d.ts` contenant les déclarations pour le module "foo".
+Vous devriez maintenant être capable d'importer `"foo"` dans votre code et il sera dirigé vers la nouvelle définition de type.
+Ensuite, compilez *et* exécutez le code pour vous assurer que votre définition de type correspond bien à ce qui se passe à l'exécution.
+
+Une fois que vous avez testé vos définitions avec du code réel, faites un [PR](#faire-une-demande-de-pull-request)
+puis suivez les instructions pour [modifier un paquet existant](#modifier-un-paquet-existant) ou
+[créer un nouveau paquet](#créer-un-nouveau-paquet).
+
+### Faire une demande de pull request
+
+Once you've tested your package, you can share it on Definitely Typed.
+
+Tout d'abord, [fork](https://guides.github.com/activities/forking/) ce dépôt, [clone](#partial-clone), installez [node](https://nodejs.org/), et lancez `npm install`. Si vous utilisez `npm` v7, vous devez ajouter le drapeau `--legacy-peer-deps` à la commande.
+
+Nous utilisons un robot pour permettre à un grand nombre de pull requests vers DefinitelyTyped d'être traitées entièrement en libre-service. Vous pouvez en savoir plus sur [pourquoi et comment ici](https://devblogs.microsoft.com/typescript/changes-to-how-we-manage-definitelytyped/). Voici une référence pratique montrant le cycle de vie de pull request à DT :
+
+<img src="https://github.com/DefinitelyTyped/dt-mergebot/blob/master/docs/dt-mergebot-lifecycle.svg">
+
+#### Partial clone
+
+<details>
+<summary>Vous pouvez cloner l'ensemble du dépôt <a href='https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository'>comme d'habitude</a>, mais il est volumineux et comprend un énorme répertoire de paquets de type.</summary>
+
+Vous pouvez cloner l'ensemble du dépôt [comme d'habitude](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository), mais il est volumineux et comprend un énorme répertoire de paquets de type. Cela prendra du temps à cloner et peut s'avérer inutilement lourd.
+
+Pour un clone plus facile à gérer qui inclut _only_ les paquets de type qui vous concernent, vous pouvez utiliser les fonctionnalités de git [`sparse-checkout`](https://git-scm.com/docs/git-sparse-checkout), [`--filter`](https://git-scm.com/docs/git-rev-list#Documentation/git-rev-list.txt---filterltfilter-specgt), et [`--depth`](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt). Cela réduira le temps de clonage et améliorera les performances de git.
```

Comment:
Not sure this is the right word, but _only_ was still in English.

```suggestion
Pour un clone plus facile à gérer qui inclut _seulement_ les paquets de type qui vous concernent, vous pouvez utiliser les fonctionnalités de git [`sparse-checkout`](https://git-scm.com/docs/git-sparse-checkout), [`--filter`](https://git-scm.com/docs/git-rev-list#Documentation/git-rev-list.txt---filterltfilter-specgt), et [`--depth`](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt). Cela réduira le temps de clonage et améliorera les performances de git.
```

Reply:
I'm not sure either, which is why I didn't change the word.